texts/epn_paper.md

```diff
@@ -0,0 +1,87 @@
+# Let's take this discussion online
+
+## Authors (in alphabetical order)
+
+- Anton Akhmerov (Delft University of Technology)
+- Xavier Bonet-Monroig (Leiden University)
+- Valla Fatemi (Yale University)
+- Eliska Greplova (ETH Zürich)
+- Evert van Nieuwenburg (University of Copenhagen)
+- Babak Seradjeh (Indiana University)
+- Daniel Varjas (Delft University of Technology)
+
+## Introduction
+
+With the lockdowns caused by the COVID-19 pandemic researchers turn to online conferencing. While posing new challenges, this format also brings multiple advantages. We argue that virtual conferences will become a common way of scientific communication, and invite the community members to join the action.
+
+---
+
+<!-- #### COVID lockdown disrupted everyone's work, travel, and likely mean that you attended an online talk. -->
+
+Your routine has changed drastically over the course of the last couple of months due to the lockdowns following the novel coronavirus outbreak.
+You now mostly stay at home, hopefully safe and healthy.
+Your classes are now taught online, and you probably had to quickly figure out how to run them at all.
+If you do experiments, your lab is either closed or operates at a much lower capacity.
+Your travel plans are cancelled. Still, despite all this, you have likely attended a scientific presentation or have even given one yourself.
+The only difference is that it happened online, perhaps as part of a virtual conference.
+
+<!-- #### Online talks were gaining momentum already before, and for a good reason. -->
+
+Virtual conferences were gaining momentum even before the coronavirus crisis due to multiple advantages over traditional in-person gatherings.
+With the increasing concerns about greenhouse emissions, many prefer to limit their long-distance travel.
+For some community members with limited mobility, either for family, health, or budget reasons, traditional conferences are off limits altogether, and online events are the only available option.
+Online events are also free from most of the expensive and time-consuming aspects of conference organisation: finding a venue, arranging lodging and catering for the participants.
+With planning and logistics out of the way, virtual conferences can be organized much faster and can easily respond to the latest scientific developments (see a comparison of typical timelines in the figure).
+Because of being easier to attend and accommodating hundreds of participants, online events are also more inclusive, and reach an even broader audience if the talks are recorded.
```

Comment:
```suggestion
All of these changes have the important effect of making attendance and the accommodation of attendants easier, making online events more inclusive.
Finally, such events can easily reach an even broader audience if the talks are recorded and published online.
```
This sentence, particularly its introductory clause, is rather awkward.

Reply:
I like the specificity of the original text. How about: Because they are easier to attend and can accommodate hundreds of participants, online events are also more inclusive…

Reply:
```suggestion
Since online events are easier to attend and since they accommodate hundreds of participants, they are also more inclusive.
If the talks are recorded and published, they reach an even broader audience.
```